lua/nvim-treesitter/parsers.lua

```diff
@@ -2598,6 +2598,16 @@ list.templ = {
   maintainers = { "@vrischmann" },
 }
 
+list.gren = {
+  install_info = {
+    branch = "main",
+    files = { "src/parser.c", "src/scanner.c" },
+    url = "https://github.com/MaeBrooks/tree-sitter-gren",
+  },
+  experimental = true,
```

Comment:
Sorry, but we no longer accept "experimental" parsers. Either the parser is stable enough for daily use, or you'll have to wait until it is.